common/artifacts/000_wc_artifacts.txt

```diff
@@ -562,12 +668,195 @@ runeblade_unholy_1 = {
 	}
 
 	martial = 1
-	combat_rating = 5
+	combat_rating = 6
+	
+	monthly_character_prestige = 0.15
+	monthly_character_piety = 0.05
+
+	command_modifier = {
+		morale_offence = 0.05
+	}
 
 	picture = "GFX_runeblade_unholy"
 	slot = weapon
 }
 
+### One-handed weapon ###
+
+# Legendary
+felomelorn = {
+	quality = 5
+	flags = { combat_weapon sword }
+
+	active = {
+		is_adult = yes
+		is_physical_class_trigger = yes
+	}
+	allowed_gift = {
+		always = no
+	}
+
+	stacking = no
+	indestructible = yes
+
+	diplomacy = 1
+	learning = 2
+	combat_rating = 10
+
+	monthly_character_prestige = 0.8
+	monthly_character_piety = 0.35
+	
+	general_opinion	= 5
+	cult_of_sunwell_opinion = 5
+
+	picture = "GFX_felomelorn"
+	slot = weapon
+}
+serathil = {
+	quality = 5
+	flags = { combat_weapon axe }
+
+	active = {
+		is_adult = yes
+		is_physical_class_trigger = yes
+	}
+	allowed_gift = {
+		always = no
+	}
+
+	stacking = no
+	indestructible = yes
+
+	martial = 2
+	combat_rating = 25
+
+	monthly_character_prestige = 1.5
+	
+	orcish_shamanism_opinion = 10
+
+	picture = "GFX_serathil"
+	slot = weapon
+}
+sulfuras = {
+	quality = 5
+	flags = { combat_weapon axe }
+
+	active = {
+		is_adult = yes
+		is_physical_class_trigger = yes
+		OR = {
+			is_strong_trigger = yes
+			is_brawny_trigger = yes
```

Comment:
this thing don't exist

Reply:
what do you mean, don't exist? the traits for strong, brawny and giant exist, as well as the condition of large_humanoid.

Reply:
Trait exists... scripted trigger called `is_brawny_trigger `, doesn't
https://ck2.paradoxwikis.com/Scripting#Scripted_block

Reply:
ok... I fell for one of Paradox's traps again. The trait called brawny is robust in code... fixed